lib/fluent/plugin/kinesis_helper/client.rb

```diff
@@ -125,12 +127,15 @@ def setup_credentials
             credentials_options[:policy] = c.policy if c.policy
             credentials_options[:duration_seconds] = c.duration_seconds if c.duration_seconds
             credentials_options[:external_id] = c.external_id if c.external_id
+            credentials_options[:sts_endpoint_url] = c.sts_endpoint_url if c.sts_endpoint_url
             if c.sts_http_proxy and @region
```

Comment:
Should we handle the case that both of c.sts_http_proxy and c.sts_endpoint_url are set at first?

Reply:
Made the recommended change in this commit: https://github.com/awslabs/aws-fluent-plugin-kinesis/pull/186/commits/3f4e2d36a02a18aaa7f7a9ec76816583292aefbb

Reply:
Thanks!